### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by 0xtiby**

### CODE REVIEW

**Comment by 0xtiby** [comment]
@PirateCode-eth Do you want to check it ? Or I merge it

**Comment by PirateCode-eth** [comment]
You can merge it, I need more time to learn frontend code😂

File: README.md [hunk 1]
```diff
@@ -1 +1,33 @@
-# ForeverPFP Frontend
+# ForeverPFP Fronten
```
Comment (PirateCode-eth) [inline README.md:1]: typo

**Comment by 0xtiby** [comment]
> Need to confirm the PFPs images like clonex, azuki & doodles are all owned by your right? we can't use PFP images not belongs to us lol.

You are right and no I don't own them. I take this from open sea this was just for exemple. On my side I own some rektguy, chimpers, we can use. If you have some others on your side send it me I will use it

**Comment by 0xtiby** [comment]
> I can see many commits but I can get this is more like the init commit, would be better small steps by different PR in the future.

Yes it like the init commit and the bind/unbind. I can stash in one inital commit. I will make small pr next.

**Comment by PirateCode-eth** [comment]
> > Need to confirm the PFPs images like clonex, azuki & doodles are all owned by your right? we can't use PFP images not belongs to us lol.
> 
> You are right and no I don't own them. I take this from open sea this was just for exemple. On my side I own some rektguy, chimpers, we can use. If you have some others on your side send it me I will use it

then we should not use them. You can use BAYC2335, BAYC7900, MAYC2387, MACY23877 if you like, I own these apes/mutants. After we launch the testnet website, we can ask more other community holders to share their PFPs in our website if them like, can't use them without their permissions.

**Comment by 0xtiby** [comment]
Nices ones. I will use them with the one I own

**Pull request merged**